ui/component/or-asset-tree/src/index.ts

```diff
@@ -327,17 +339,35 @@ export class OrAssetTree extends subscribe(manager)(LitElement) {
                             (v) => this._onSortClicked(v as string))}
                 </div>
             </div>
-
+            
+            <div id="asset-tree-filter">
+                <or-mwc-input id="filterInput"
+                              style="width: 100%;"
+                              type="${ InputType.FILTER }"
+                              .value="${ this._filterValue }"
+                              icon="magnify" iconTrailing="${this._filterValue.length > 0 ? 'close' : ''}" 
+                              compact="true"
+                              eventOnInput="true"
+                              @or-mwc-input-changed="${ (e: OrInputChangedEvent) => {
+                                this._onFilterValueChange(e);
+                              } }">
+                              </or-mwc-input>
+            </div>
+            
             ${!this._nodes
                 ? html`
                     <span id="loading"><or-translate value="loading"></or-translate></span>`
-                : html`
+                : (this._nodes.length === 0 && this._isFiltering
```

Comment:
Can this be combined with the first check (i.e. `!this._nodes || this._isFiltering`)

Reply:
(y)